File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/PlusCode.java

package restaurant.sa.com.sarestaurant.appview.restaurant.model;

import com.google.gson.annotations.SerializedName;

public class PlusCode {

    @SerializedName("compound_code")
    private String compoundCode;
    @SerializedName("global_code")
    private String globalCode;

    public String getCompoundCode() {
        return compoundCode;
    }

    public void setCompoundCode(String compoundCode) {
        this.compoundCode = compoundCode;
    }

    public String getGlobalCode() {
        return globalCode;
    }

    public void setGlobalCode(String globalCode) {
        this.globalCode = globalCode;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/restaurant/model/PlusCode.kt
package restaurant.sa.com.sarestaurant.appview.restaurant.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Entity(tableName = "PlusCode")
data class PlusCode(
        @SerializedName("compound_code")
        @Expose
        @ColumnInfo(name = "compoundCode") var compoundCode: String?,
        @SerializedName("global_code")
        @Expose
        @ColumnInfo(name = "globalCode") var globalCode: String?)